server/server.go

```diff
@@ -0,0 +1,37 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/sungatuly22/tournament-service/db"
+)
+
+type Server struct {
```

Comment:
Once you change CreateUser and other functions to methods add here `UserStorage` field